theorems/T000724.md

```diff
@@ -0,0 +1,19 @@
+---
+uid: T000724
+if:
+  and:
+  - P000003: true
+  - P000030: true
```

Comment:
```suggestion
  - P000030: true
  - P000003: true
```

Reply:
am I blanking on a style decision we made here? I usually put them in order by ID I think.

Reply:
Not a codified style guideline, but we usually put the most important property first, which is how people usually phrase theorems when written in a paper.  In this case, it seems better to put paracompact first, followed by the corresponding separation axiom.  (So in general we could have "paracompact Hausdorff" or "paracompact regular" instead of "Hausdorff paracompact", etc.)

Reply:
I don't feel strongly about this either way, so I have no problem merging. If we want this done consistently, we should probably document in the wiki and discuss in a community call someday.